tests/run_tests.sh

```diff
@@ -23,9 +23,9 @@ function before_test() {
    vagrant ssh-config | awk '
```

Comment:
Vagrant will create a inventory with these settings for you if you use the `ansible` provisioner, even with a no-op playbook. Using that inventory (or copying its entry to another inventory) would be cleaner than extracting the information ourselves.
```
$ cat Vagrantfile
# -*- mode: ruby -*-
# vi: set ft=ruby :

Vagrant.configure("2") do |config|
  config.vm.box = "bento/ubuntu-16.04"
  config.vm.provision "ansible" do |ansible|
    ansible.playbook = "noop.yml"
  end
end
$ cat noop.yml
---
- hosts: all
$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
    ··· snip ···
==> default: Running provisioner: ansible...
    default: Running ansible-playbook...

PLAY [all] *********************************************************************

TASK [setup] *******************************************************************
ok: [default]

PLAY RECAP *********************************************************************
default                    : ok=1    changed=0    unreachable=0    failed=0   

$ cat .vagrant/provisioners/ansible/inventory/vagrant_ansible_inventory
# Generated by Vagrant

default ansible_ssh_host=127.0.0.1 ansible_ssh_port=2222 ansible_ssh_user='vagrant' ansible_ssh_private_key_file='/private/tmp/testvagrant/.vagrant/machines/default/virtualbox/private_key'
```